src/ICSharpCode.SharpZipLib/Zip/ZipFile.cs

```diff
@@ -1620,7 +1620,7 @@ private void AddUpdate(ZipUpdate update)
 		{
 			contentsEdited_ = true;
 
-			int index = FindExistingUpdate(update.Entry.Name);
+			int index = FindExistingUpdate(update.Entry.Name, true);
```

Comment:
I try to add labels for all booleans to make the code more descriptive (without relying on an IDE).
```suggestion
			int index = FindExistingUpdate(update.Entry.Name, isEntityName: true);
```